docs/components/compact.md

```diff
@@ -428,6 +428,10 @@ Flags:
                                 Prometheus relabel-config syntax. See format
                                 details:
                                 https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config
+      --web.route-prefix=""     Prefix for API and UI endpoints. This allows
```

Comment:
This is great, but let's make sure this flag is handled and not registered and not used, by accident (: